components/esp_lvgl_port/examples/rgb_lcd/main/idf_component.yml

```
@@ -0,0 +1,8 @@
+dependencies:
+  idf: ">=4.4"
```

```suggestion
dependencies:
  idf: ">=5.0"
```
5.0 for RGB support?